packages/amplify_core/pubspec.yaml

```diff
@@ -14,6 +14,8 @@ dependencies:
   amplify_storage_plugin_interface: ^0.0.1-dev.3
   amplify_analytics_plugin_interface: ^0.0.1-dev.3
   amplify_auth_plugin_interface: ^0.0.1-dev.3
+  amplify_datastore_plugin_interface:
+    path: ../amplify_datastore_plugin_interface
```

Comment:
We need to change this local path.  This will impact our release sequence.

Reply:
Agree. Should we make the change in a separate PR once we are done testing? This will allow `flutter pub get` to work during local testing.

Reply:
you actually shouldn't need to do `flutter pub get` since melos should be doing it for you.

Reply:
validated that removing the path doesn't break and melos bootstrap is enough. will be pushing this change in this PR.